README.md

```diff
@@ -7,15 +7,59 @@
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![Griptape Discord](https://dcbadge.vercel.app/api/server/gnWRz88eym?compact=true&style=flat)](https://discord.gg/gnWRz88eym)
 
-**Griptape** is a modular Python framework for building AI-powered applications that connect securely to your enterprise data and APIs. It offers developers the ability to maintain control and flexibility at every step.
+Griptape is a modular Python framework for building AI-powered applications that connect securely to your enterprise data and APIs. It offers developers the ability to maintain control and flexibility at every step.
 
-**Build AI Apps**: Easily compose apps in Python with modular structures and ready-made tools. Use built-in drivers to connect to whichever LLMs and data stores you choose.
+## 🛠️ Core Components
 
-**Control Data Access**: Connect securely to data sources with granular access controls, ensuring LLMs stay focused on the information that matters.
+### 🏗️ Structures
 
-**Scale With Your Workload**: Easily deploy and run apps in the cloud, where your data lives. Process data ahead of time or vectorize it on the fly.
+- 🤖 **Agents** consist of a single Task.
+- 🔄 **Pipelines** organize a sequence of Tasks so that the output from one Task may flow into the next.
+- 🌐 **Workflows** configure Tasks to operate in parallel.
 
-Using Griptape, you can securely integrate with your internal data stores and APIs. You get to control what data goes into the prompt, and what the LLM is allowed to do with it. 
+### 📝 Tasks
```

Comment:
should we move Tasks to the top since they are "the core building block"? or is the intention to have Structures be at the top since its the thing that people actually use

Reply:
Yeah went back and forth on this. They're kind of co-dependent but Structures feel more important.

Reply:
makes sense. maybe we slightly differentiate/emphasize structures then? not sure of a good way to do that